nginx.conf

```diff
@@ -138,6 +138,7 @@ server {
 		# Plex Headers
 		proxy_set_header X-Plex-Client-Identifier $http_x_plex_client_identifier;
 		proxy_set_header X-Plex-Container-Size $http_x_plex_container_size;
+		proxy_set_header X-Plex-Container-Start $http_x_plex_container_start;
```

Comment:
```log
Jul 11, 2022 17:41:19.341 [0x7f106b0e0b38] WARN - [Req#c3e9] Missing X-Plex-Container-Size header. This will fail with status code 400 in the future.
```